Model/Payment/FastPayment.php

```diff
@@ -419,6 +419,11 @@ public function getConfigData($field, $storeId = null)
             $storeId = $this->getStore();
         }
         $path = 'fast_integration/fast/' . $field;
+
+        if ('title' === $field) {
+            $path = 'payment/' . $this->getCode() . '/' . $field;
+        }
+
```

Comment:
Could you combine these like this?

```suggestion
        $path = ('title === $field) ? 'payment/' . $this->getCode() . '/' . $field : 'fast_integration/fast/' . $field;
```